admin_pages/messages/templates/ee_msg_editor_active_context_element.template.php

```diff
@@ -36,14 +40,14 @@
     <span class="spinner"></span>
     <div class="activate_context_on_off_toggle_container">
         <span id="on-off-nonce-<?php echo esc_attr($context); ?>" class="hidden">
-            <?php echo $nonce; // already escaped ?>
+            <?php echo wp_kses($nonce, AllowedTags::getWithFormTags()); ?>
         </span>
         <span class="ee-on-off-toggle-label">
-            <?php echo $is_active ? $active_message : $inactive_message; // vars already escaped ?>
+            <?php echo ($is_active ? wp_kses($active_message, AllowedTags::getAllowedTags()) : wp_kses($inactive_message, AllowedTags::getAllowedTags())); ?>
```

Comment:
plz remove unnecessary brackets and add some line breaks:

```suggestion
            <?php
             echo $is_active 
          		? wp_kses($active_message, AllowedTags::getAllowedTags())
          		: wp_kses($inactive_message, AllowedTags::getAllowedTags()); 
          	?>
```

Reply:
Fixed.